sklearn/linear_model/tests/test_base.py

```diff
@@ -57,8 +56,10 @@ def test_linear_regression():
 
 @pytest.mark.parametrize("array_constr", [np.array, sparse.csr_matrix])
 @pytest.mark.parametrize("fit_intercept", [True, False])
-def test_linear_regression_sample_weights(array_constr, fit_intercept):
-    rng = np.random.RandomState(0)
+def test_linear_regression_sample_weights(
+    array_constr, fit_intercept, global_random_seed
+):
+    rng = check_random_state(global_random_seed)
```

Comment:
updated for consistency: `check_random_state` is used in some functions, but not all